templates/governance/group-team.hbs

```diff
@@ -52,5 +52,23 @@
                 </div>
             {{/each}}
         </div>
+        {{#if team.alumni}}
+            <header class="pb3 pt3">
+                <h2 class="f2">{{fluent "governance-alumni-header"}}</h2>
+            </header>
+            <p>{{fluent "governance-alumni-thanks"}}</p>
+            <div class="flex flex-column flex-row-l flex-wrap-l justify-start">
+                {{#each team.alumni as |member|}}
+                <div class="w-100 w-33-l mb3 flex flex-row items-center">
+                    <a class="mr3 w2 h2 flex-no-shrink" href="https://github.com/{{member.github}}">
```

Comment:
Okay I tested this out locally, and I think if the images were slightly bigger overall it would better for desktop but particularly mobile.

<img width="308" alt="Screenshot 2020-03-07 at 12 21 25" src="https://user-images.githubusercontent.com/4464295/76142456-45af3200-606e-11ea-90a6-85c5aeb8a4e7.png">


```suggestion
                    <a class="mr3 w3 h3 flex-no-shrink" href="https://github.com/{{member.github}}">
```

Reply:
I'm not sure if that's too much emphasis.

Reply:
I disagree that it's putting too much emphasis, it's takes up a quarter of the area that current member's image does, that's a pretty clear visual hierarchy, I don't think you could ever confuse the two sections.

For me a lot of the images are illegible at the current size. This change makes the image a similar size to the avatar in a top level GitHub comment, which is where most people see their avatar, so it's more likely to be legible.

I also don't think I agree with the motivation that alumni should not have too much emphasis. It needs to be clearly distinguished from the current members sections, but not necessarily de-emphasised. These people's work was important, we shouldn't de-emphasise that.

Reply:
Hmm, the big avatar with the small name doesn't look too great... We could add the GitHub username as well, but it emphasizes the "GitHub label" problem even more:

![2020-03-10--12-32-49](https://user-images.githubusercontent.com/2299951/76308664-8eaff200-62cb-11ea-94e1-e44a9689cc61.png)

Reply:
I like that approach. It's more consistent. I can make a PR later switching the GitHub label to an image.

Reply:
Ok updated the PR.